service/src/test/resources/core-case-data-store-api/callback-request.json

```diff
@@ -6,6 +6,7 @@
     "case_type_id": "Shared_Storage_DRAFTType",
     "security_classification": "PUBLIC",
     "case_data": {
+      "gateKeeperEmail": "FamilyPublicLaw+gatekeeper@gmail.com",
```

Comment:
Don't we need to add same to `src/test/resources/core-case-data-store-api/populated-case-details.json`?